### ISSUE TITLE
[Idea] WebGL Support?

### ISSUE THREAD

**Issue by Fancy2209** [TooTallNate/nx.js#66]
Some kind of Hardware Acceleration with WebGL or WebGPU support would be neat for this.
https://developer.mozilla.org/en-US/docs/Web/API/WebGL_API
https://developer.mozilla.org/en-US/docs/Web/API/WebGPU_API
Nice work by the way!

**Comment by TooTallNate** [comment]
Definitely! I've been focusing on the 2D canvas context so far, but WebGL context may come after that. Open to pull requests if you'd like to get involved as well.

**Comment by Fancy2209** [comment]
> Definitely! I've been focusing on the 2D canvas context so far, but WebGL context may come after that. Open to pull requests if you'd like to get involved as well.

I don't have the knowledge to contribute unfortunately, I would if I I could!

**Comment by TooTallNate** [comment]
Follow #131 for an initial take at this.

**Comment by TooTallNate** [comment]
Implemented `webgl2` context in https://github.com/TooTallNate/nx.js/pull/390.
